scripts/serve-local.sh

```diff
@@ -19,6 +19,7 @@ docker run \
     --volume "$PWD/static.config.js:/app/static.config.js" \
     --volume "$PWD/.babelrc:/app/.babelrc" \
     --volume "$PWD/yarn.lock:/app/yarn.lock" \
+    --env "FEEDBACK_API=https://coa-test-form-api.herokuapp.com/process/" \
```

Comment:
Should this be set as an environment variable if it is the same for all the environments?

Reply:
I'm thinking we'll have different endpoints in different environments (with prod having more rigorous validation). We probably don't want issues being created in dev/staging as is currently the case, and I haven't thought through what the long term plan is, but this leaves it open. Also cool with changing -- lemme know what you're thinking.